solr/core/src/test/org/apache/solr/core/TestConfigSetService.java

```diff
@@ -77,24 +72,31 @@ public void testConfigSetServiceOperations() throws IOException {
     byte[] testdata = "test data".getBytes(StandardCharsets.UTF_8);
 
     Path configDir = createTempDir("testconfig");
-    Files.createFile(configDir.resolve("solrconfig.xml"));
-    Files.write(configDir.resolve("file1"), testdata);
-    Files.createFile(configDir.resolve("file2"));
-    Files.createDirectory(configDir.resolve("subdir"));
-    Files.createFile(configDir.resolve("subdir").resolve("file3"));
+    String solrConfigXml = "solrconfig.xml";
+    Files.createFile(configDir.resolve(solrConfigXml));
+    String file1 = "file1";
+    Files.write(configDir.resolve(file1), testdata);
+    String file2 = "file2";
+    Files.createFile(configDir.resolve(file2));
+    String subDirPath = "subdir";
+    String subdir = subDirPath + "/";
+    Files.createDirectory(configDir.resolve(subDirPath));
+    String file3 = subdir + "file3";
+    Files.createFile(configDir.resolve(subDirPath).resolve("file3"));
+    String file4 = subdir + "file4";
 
```

Comment:
Ooops.
When I said to keep these changes from your trial PR, I wasn't paying attention at all.  I glanced I guess and thought you were converting from java.io.File to the Path API -- quite wrong and embarrassing on my part.  I think the test was easier to read before, honestly, because the assertions had the strings plainly without having to decipher what each variable added means.  **But it's fine; professionals can disagree.**

Reply:
yea I'll clean this up. I'm not really a fan of how it turned out.